kafka-connect-s3/src/main/java/io/confluent/connect/s3/TopicPartitionWriter.java

```diff
@@ -349,6 +349,11 @@ private RecordWriter getWriter(SinkRecord record, String encodedPartition) throw
       return writers.get(encodedPartition);
     }
     String commitFilename = getCommitFilename(encodedPartition);
+    log.debug(
+        "Creating new writer encodedPartition={} filename={}",
```

Comment:
In similar log lines I've found that is useful to enclose the values in single quotes `'`. Makes reading of such lines easier when the value is empty. Low priority.